team-validator.js

```diff
@@ -253,16 +253,15 @@ class Validator {
 				if (!set.moves[i]) continue;
 				let move = tools.getMove(Tools.getString(set.moves[i]));
 				if (!move.exists) return ['"' + move.name + '" is an invalid move.'];
-				set.moves[i] = move.name;
```

Comment:
I'll have to double-check this since it's been so long but my recollection was that it was the ruleset that was choking the second time around because the validator changed the moves from ids to names.